docs/notebooks/hydrotel.ipynb

```diff
@@ -0,0 +1,326 @@
+        "\n",
+        "By default, all output options are turned off, so the required outputs need to be turned on too if it's not already the case."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "e5cc7424-c691-4c2b-b5a4-4ccfe01372b1",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# The options only need to be those that differ from the defaults or those on-disk. These will generally include the simulation dates and output requested.\n",
+        "simulation_options = {\"DATE DEBUT\": \"2001-01-01\", \"DATE FIN\": \"2001-12-31\", \"FICHIER STATIONS METEO\": \"meteo/SLNO_meteo_GC3H.nc\", \"PAS DE TEMPS\": 24}\n",
+        "output_options = {\"TRONCONS\": 1, \"DEBITS_AVAL\": 1}\n",
+        "ht = Hydrotel(project_folder / project_name, default_options=False, simulation_options=simulation_options, output_options=output_options)\n",
+        "\n",
+        "print(f\"Simulation name, taken from 'projet.csv': '{ht.simulation_name}'\\n\")\n",
+        "print(f\"Project configuration: '{ht.project_options}'\\n\")\n",
+        "print(f\"Simulation configuration: '{ht.simulation_options}'\\n\")\n",
+        "print(f\"Output configuration: '{ht.output_options}'\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "b839b2b9-0b9f-46f1-b684-08879e0ec6e1",
+      "metadata": {
+        "editable": true,
+        "slideshow": {
+          "slide_type": ""
+        },
+        "tags": []
+      },
+      "source": [
+        "## Validating the meteorological data\n",
+        "A few basic checks will be automatically performed prior to executing the model, but a user might want to perform more advanced health checks (missing values, unrealistic meteorological inputs, etc.). This is possible through the use of `xhydro.utils.health_checks`. Consult [the 'xscen' documentation](https://xscen.readthedocs.io/en/latest/notebooks/3_diagnostics.html#Health-checks) for the full list of possible checks.\n",
+        "\n",
+        "In this example, we'll:\n",
+        "- Make sure that there is no missing data.\n",
+        "- That there are no abnormal meteorological values or sequence of values. Some checks here will fail, to showcase what those would look like."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "06cb6eb2-366a-4f9a-be37-ecbe6ebbb833",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "health_checks = {\"raise_on\": [],  # If an antry is not here, it will warn the user instead of raising an exception.\n",
```

Comment:
```suggestion
        "health_checks = {\"raise_on\": [],  # If an entry is not here, it will warn the user instead of raising an exception.\n",
```